code/modules/client/client defines.dm

```diff
@@ -12,6 +12,7 @@
 		//OTHER//
 		/////////
 	var/datum/preferences/prefs = null
+	var/skip_antag = FALSE //Decline to be selected as a game mode antagonist.
```

Comment:
Declined

Reply:
I'm gonna change the wording of this comment entirely actually